### DISCUSSION TITLE
Icons used in Flex Queue Stats

### DISCUSSION THREAD

**Discussion by Fazle-Jahangir** [twilio-labs/paste#3121] [Request: Icon] [answered]
[Redacted]

Non-public information

**Comment by loreina** [comment, marked as answer]
Hi @Fazle-Jahangir,

For SMS, you could use the `Chat` or `SMSCapable` icon, though keep in mind that the latter is part of a set of "capable" icons including MMS, Fax, Voice, etc. 

For the folder icon that indicates "Everyone", you could use `Users` icon already in Paste.

For the unknown channels icon, we don't currently have something similar in Paste. In a previous discussion https://github.com/twilio-labs/paste/discussions/2134#discussioncomment-2159072, we've recommended using a general icon like a call icon when the call is known. You could consider a general icon here for channels or select one of the current icons like `Chat` for when it's unknown.

If you'd rather add a new icon, please provide a link to the Streamline icon and [submit a dependency request](https://airtable.com/shrvMvL3BG6Zm0f9l). In the future for icon requests, please use the "Request: Icon" category as it's got a form with details so our team can consider the request and reduce back and forth.

Thanks!

**Reply by Fazle-Jahangir** [reply]
Hi @loreina 
Thanks for the recommendations. There's a few icon options for the **SMS** channel as you stated, it is the **messaging** channel I am struggling to decide the icon to use because the messaging icon used in Flex 1.0 is now used for something else (called _SMS capable_) in UI 2.0. Image below shows how the _SMS capable_ icon is used for messaging in Flex 1.0. You recommended  the _chat_ icon for the SMS channel, do you think it's a good fit instead for the messaging channel? or perhaps if it's ok to continue using _SMS capable_ for messaging?
<img width="256" alt="Screenshot 2023-03-24 at 11 28 13" src="https://user-images.githubusercontent.com/100706574/227915816-19c51b3b-a037-46ae-a34f-f6993cc233ed.png">

**Reply by loreina** [reply]
Hi @Fazle-Jahangir,

Do you have concerns with switching to the `Chat` icon for messaging? That's also the icon recommended for use in the Conversations UI Kit to indicate messaging: https://paste.twilio.design/components/minimizable-dialog#basic-minimizable-dialog.

**Reply by Fazle-Jahangir** [reply]
[Redacted]

Non-public information

**Reply by loreina** [reply]
Makes sense! We can add a folder icon—can you share a link to a [Streamline icon](https://www.streamlinehq.com/)?

As for an icon for channel type unknown, can you provide a link to a Streamline icon so we can understand what glyph you're looking to add? My concern is that the icon could be very use case specific, whereas text paired with a generic icon could reduce the cognitive load on the user to understand the icon's meaning.

Once the dependency request is in and we've decided which icons will be added to Paste, our team will add the work to our backlog for Q2. Thanks!

**Reply by Fazle-Jahangir** [reply]
[Redacted]

Non-public information

**Reply by loreina** [reply]
Hi @Fazle-Jahangir,

I spoke with the team and have a few design suggestions:

1. Since the queue is part of a disclosure that already has an icon, we'd recommend removing the folder icon and aligning the queue items that aren't in a folder with the disclosure icon.
2. Since the product is for conversations, we'd recommend using the `Chat` icon for generic channels rather than introducing an icon to indicate "unknown". 

I made a mockup for visual reference: https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?node-id=2410-428471.

Let me know what you think and if you have any questions!

**Reply by Fazle-Jahangir** [reply]
Thanks for the mockup @loreina 
I'm onboard with the suggestions!
